### PR TITLE
Created Tower of Hanoi in PHP #2513

### PR DESCRIPTION
**Pull request by aryangulati**
Fixes #2513

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Created Tower Of Hanoi Using PHP

#### Languages Used:
- PHP

#### Files Added:
- Tower_Of_Hanoi/Tower_Of_Hanoi.php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by MastersAbh** [comment]
@aryangulati please add issue number in the problem description

**Comment by aryangulati** [comment]
What is PR checklist and where can i get it my issue no. Is 2513 solved and PR no. Is 2522

**Comment by asha15** [comment]
Can i get a link to your issue?

**Comment by aryangulati** [comment]
https://github.com/jainaman224/Algo_Ds_Notes/pull/2513

File: Tower_Of_Hanoi/Tower_Of_Hanoi.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// Tower of Hanoi 
+// Recursive Function to solve Tower of Hanoi 
+function towerOfHanoi($n, $from_rod, $to_rod, $aux_rod) { 
+	//base case 
+	if ($n === 1) { 
+	echo ("Move disk 1 from rod $from_rod to rod $to_rod \n"); 
+	return; 
+	} 
+	
+	towerOfHanoi($n-1, $from_rod, $aux_rod, $to_rod);
```
Comment (singh-shreya6) [inline Tower_Of_Hanoi/Tower_Of_Hanoi.php:11]: Please add space around - in n-1

File: Tower_Of_Hanoi/Tower_Of_Hanoi.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+function towerOfHanoi($n, $from_rod, $to_rod, $aux_rod) { 
+	//base case 
+	if ($n === 1) { 
+	echo ("Move disk 1 from rod $from_rod to rod $to_rod \n"); 
+	return; 
+	} 
+	
+	towerOfHanoi($n-1, $from_rod, $aux_rod, $to_rod); 
+	echo ("Move disk $n from rod $from_rod to rod $to_rod \n"); 
+	towerOfHanoi($n-1, $aux_rod, $to_rod, $from_rod);
```
Comment (singh-shreya6) [inline Tower_Of_Hanoi/Tower_Of_Hanoi.php:13]: Please add space around -

File: Tower_Of_Hanoi/Tower_Of_Hanoi.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	
+	towerOfHanoi($n-1, $from_rod, $aux_rod, $to_rod); 
+	echo ("Move disk $n from rod $from_rod to rod $to_rod \n"); 
+	towerOfHanoi($n-1, $aux_rod, $to_rod, $from_rod); 
+} 
+// number of disks 
+$n = 4; 
+// A, B and C are names of rods 
+towerOfHanoi($n, 'A', 'C', 'B'); 
+//OUTPUT:
```
Comment (singh-shreya6) [inline Tower_Of_Hanoi/Tower_Of_Hanoi.php:19]: Please change these single line comments into one multi line comment

**Comment by somya-kapoor** [comment]
@aryangulati was this issue assigned to you?

**Comment by singh-shreya6** [comment]
No updates from several days. Closing this PR, please raise a new one if you want to work on it again.

**Pull request closed without merging**